eval_engines/ngspice/ngspice_inputs/yaml_files/zhenxin_s_fc.yaml

```
@@ -0,0 +1,26 @@
+database_dir: "Autockt/checkpoint/db/Zhenxin_S_FC_ngspice"
+dsn_netlist: "eval_engines/ngspice/ngspice_inputs/netlist/Zhenxin_S_FC.cir"
+num_process: 1
+
+params:
+  w_m12:  !!python/tuple [1, 100, 100]
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)